SampleApp/build.gradle

```diff
@@ -14,4 +14,4 @@ task build {
      }
 }
 }
-build.dependsOn(':src:AirshipBindings.NETStandard:build')
+build.dependsOn(':pkg')
```

Comment:
This is to force the sample app to wait to build until packages are available. If you don't do that you can get into a state where you're unable to create local packages because SampleApp is failing to build when it can't find them.